_posts/2018-11-16-using-vault-with-strimzi.md

```diff
@@ -166,7 +171,7 @@ kubectl create secret -n ${strimziNamespace} generic ${clusterName}-cluster-ca \
 
 # Certificate
 kubectl create secret -n ${strimziNamespace} generic ${clusterName}-cluster-ca-cert \
-  --from-file=ca.crt=intermediate.cert.pem \
+  --from-file=ca.crt=intermediate.chain.pem \
```

Comment:
what is done here let me think that you are providing a certs chain to Strimzi as Cluster CA cert which sounds strange, at least to me :-)
Now Strimzi is going to sign the autogenerated ZK and K brokers/nodes certificates using this PEM file containing a chain. To be honest I have never done something like that. Isn't it enough to generate the Secret passing the intermediate CA cert and the Strimzi will sign brokers/nodes certs with that one? @tombentley @scholzj wdyt?

Reply:
If it works? Keep in mind that this is just the public key with the chain. I assume there is no chain for the private key which should IMHO make it ok.

Reply:
Yeah I said a stupid one. Certs will be signed by related intermediate CA private key not by public certs in the chain.